packages/modal/src/Modal.stories.tsx

```diff
@@ -215,3 +271,20 @@ console.log(greeting('World'));
     </div>
   );
 }
+
+export function TooltipTrigger(args: ModalProps) {
```

Comment:
Should also either have the tooltip open immediately, or write a `play` function so this gets checked in Chromatic